pyproject.toml

```diff
@@ -9,6 +9,8 @@ readme = "README.md"
 python = "^3.9"
 tiktoken = ">=0.5.2"
 openai = ">=1.9.0"
+anthropic = "^0.28.0"
+google-generativeai = "^0.6.0"
```

Comment:
can you please make it through:
```
[build-system]
requires = ["setuptools", "wheel"]
build-backend = "setuptools.build_meta"

[project]
name = "python-lib"
version = "0.1.0"
description = "A sample Python library"
dependencies = [
    # Your main dependencies
]

[project.optional-dependencies]
dep1 = [
    "some_optional_dependency1",
    "some_optional_dependency2",
]
all = [
    "some_optional_dependency1",
    "some_optional_dependency2",
    "another_optional_dependency",
]
```

Reply:
![image](https://github.com/LamoomAI/flow-prompt/assets/92877003/2606898e-b0c9-4fc2-bae8-d1cd249f3299)

Do you mean like it was done with `flow-prompt`?